client/js/lounge.js

```diff
@@ -342,10 +343,18 @@ $(function() {
 				data.msg
 			]);
 
-		if (data.msg.self) {
+		// Unread marker is "hidden" (moved to bottom) if message comes from own
+		// user or if the channel is both active and focused
+		if (data.msg.self || (channel.hasClass("active") && document.hasFocus())) {
 			container
 				.find(".unread-marker")
 				.appendTo(container);
+
+			// Channel is marked as read on the server
+			socket.emit(
```

Comment:
For performance reason? This doesn't strike me as such a terrible thing considering `open` is a small, fixed-sized event.
In terms of behavior, this is natural as a client acknowledging it read a message is more accurate than making this assumption from the server.

Any alternative in mind?